tsconfig.base.json

```diff
@@ -31,6 +31,7 @@
       "es2018",
       "es2019",
       "es2020",
+      "es2022",
```

Comment:
It looks like the lowest version of Node we support is 16, and 16.13+ does support `Error.cause` so I think this is ok? 

See: https://node.green/#ES2022-features-Error-cause-property-Error-has-cause